python/vyos/ifconfig/ethernet.py

```diff
@@ -258,6 +259,21 @@ def set_rps(self, state):
         # send bitmask representation as hex string without leading '0x'
         return self.set_interface('rps', rps_cpus)
 
+    def set_rps(self, state):
+        rfs_flow = 0
+        global_rfs_flow = 0
+        queues = glob(f'/sys/class/net/{ifname}/queues/rx-*')
+        if state:
+            global_rfs_flow = 32768
+            rfs_flow = 32768/queues
```

Comment:
Why not reference global_rfs_flows here?

Reply:
done